src/components/App.js

```diff
@@ -17,12 +18,16 @@ const App = () => {
     }
   }
 
-  const addTodo = () => setTodos([...todos, todo])
+  const addTodo = () => {
+    const value = { title: todo, status: 'pending' }
+    setTodos([...todos, value])
```

Comment:
Para hacerlo un poco más declarativo, podés cambiar `value` por un nombre más específico

```suggestion
    const newTodo = { title: todo, status: 'pending' }
    setTodos([...todos, newTodo])
```